cmd/rhoas/main.go

```diff
@@ -128,3 +137,25 @@ func initConfig(f *factory.Factory) {
 		os.Exit(1)
 	}
 }
+
+// check if the config file is located in the old default location
+// if so, move it to the new location
+// TODO: Remove this function after 0.28.0 is released.
```

Comment:
I would remove this and transform that to the issue (possibly not instantly removed since we do not control when people update etc.)

Reply:
Since login to CLI is required everyday I would question if migration would be critical thing to do, but let's have it :D

Reply:
Here's a scenario:

1. User logs in - sees there is an update
2. Downloads the update
3. They have to log in again!!

Another scenario:

User is logged in and using their Kafka which is saved to their config. They update, have to log in and their Kafka is no long er saved as the current one.

These are not drastic things to haver happen. But I do want to build these good habits early.